ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -208,11 +212,15 @@
 {%                         if router_bgp.peer_groups[router_bgp.vrfs[vrf].neighbors[neighbor].peer_group].bfd is arista.avd.defined(true) %}
 {%                             set inherited_vrf.bfd = "Inherited from peer group " ~ router_bgp.vrfs[vrf].neighbors[neighbor].peer_group %}
 {%                         endif %}
+{%                         if router_bgp.peer_groups[router_bgp.vrfs[vrf].neighbors[neighbor].peer_group].shutdown is arista.avd.defined(true) %}
```

Comment:
this should also inherit (after the vrf specific peer_group config) from the general peer_group config. That is how EOS does it.

Reply:
Ok, got it.

Reply:
I think I added the inheritance in line 216.

Reply:
SEDI-shut is the testcase for this.